src/towncrier/newsfragments/84.bugfix.rst

```diff
@@ -0,0 +1 @@
+Stop showing tracebacks for known configuration errors.
```

Comment:
```suggestion
Configuration errors found during command line execution now trigger a message to stderr and no longer show a traceback.
```

Reply:
Agreed, this is a much better description.  Thanks.